setup.py

```diff
@@ -39,3 +39,27 @@
     },
     package_data={'beastling': ['data/*']},
 )
+
+setup(
```

Comment:
Wouldn't it be simpler to only add another `console_scripts` entry point to the BEASTling setup?